charts/kubearchive/values.yaml

```diff
@@ -49,18 +40,29 @@ sink:
   # If true, OpenTelemetry instrumentation will be enabled for the sink
   observability: false
 
+# values used to create the operator
 operator:
+  name: "{{ .Release.Name }}-operator"
   # NOTE - Helm does not resolve the `ko` path. The path needs to be set on the `helm install` command.
   # For example:
   # helm install -n default <chart-name> charts/kubearchive \
   # --set-string operator.image=$(ko build github.com/kubearchive/kubearchive/cmd/operator)
   image: ko://github.com/kubearchive/kubearchive/cmd/operator
+  leaderElection: "{{ tpl .Values.operator.name . }}-leader-election"
```

Comment:
Why are all these being added?

Reply:
What do you mean? I extracted this value to the `values.yaml` to avoid repeating `{{ tpl .Values.operator.name . }}-leader-election` in the 6 places that it appears now in the files `leader_election_role.yaml` and `leader_election_role_binding.yaml`.

Reply:
I've tried to comply with the conventions described in [here](https://kubearchive.github.io/kubearchive/main/contributors/naming-conventions.html)

Reply:
I have to say as I look at this change that I don't like it.  I just don't see the value of extracting all these names individually to the `values.yaml` file.  The only part of these names that I think will ever change is the first part, and I don't think we'll ever change the name values defined in `values.yaml`.

I also don't see the need to put the additional settings (that weren't templated before) in the`values.yaml` either.  We don't have to have everything there.

I think we should try to keep `values.yaml` small in general,and only include things there that are likely to change.